src/hotspot/share/utilities/globalDefinitions_gcc.hpp

```diff
@@ -171,4 +173,8 @@ inline int g_isfinite(jdouble f)                 { return isfinite(f); }
 //
 #define ATTRIBUTE_ALIGNED(x) __attribute__((aligned(x)))
 
+// Converts any type T to a reference type.
+template<typename T>
+std::add_rvalue_reference_t<T> declval() noexcept;
```

Comment:
Why is this in the gcc-specific file?  How can this even work? The use in vmStructs.cpp is in shared code.

Reply:
Oh, I pushed an incomplete change, sorry. I'm done now.